week-5/C-promises/exercise.js

```diff
@@ -60,6 +76,10 @@ function exercise4() {
 
 function exercise5() {
   // Write your implementation here
+  var promise5 = Promise.resolve("Hello Promises");
```

Comment:
Nice, nice and simple 🙂 There's also another way which you can create a resolved promise, do you know what it is? Hint: it's a bit longer, but is more flexible in how you could use it.